streampark-console/streampark-console-service/src/test/java/org/apache/streampark/console/core/service/SettingServiceTest.java

```diff
@@ -98,4 +98,14 @@ void checkEmailTest() {
     ResponseResult result = settingService.checkEmail(senderEmail);
     Assertions.assertEquals(result.getStatus(), 200);
   }
+
+  @Test
+  void checkDockerTest() {
+    DockerConfig dockerConfig = new DockerConfig();
+    dockerConfig.setAddress("https://hub.docker.com/v2/users/login");
+    dockerConfig.setUserName("username");
+    dockerConfig.setPassword("password");
+    dockerConfig.setNamespace("");
```

Comment:
dockerConfig.setNamespace("streampark");

Reply:
I don't think there is any use in adding this test now, I suggest adding it together after adding the e2e test